oem/ibm/configurations/bios/com.ibm.Hardware.Chassis.Model.Bonnell/enum_attrs.json

```diff
@@ -64,15 +64,15 @@
             "attribute_name": "hb_predictive_mem_guard",
             "possible_values": ["Enabled", "Disabled"],
             "default_values": ["Enabled"],
-            "helpText": "Enable or Disable Predictive Guard for Memory Errors.",
-            "displayName": "Predictive Memory Guard"
+            "helpText": "The Hypervisor dynamically removes memory pages from memory that detected predictive failures. If this option is disabled, dynamic memory deallocation will not occur for the predictive memory failures.",
```

Comment:
Would it be better to write "The hypervisor dynamically removes the memory pages which were identified as predictive failures" ?

Reply:
I have written what was agreed by Jim.